Main.java

```diff
@@ -0,0 +1,24 @@
+import exceptions.DuplicateProductException;
+import exceptions.ProductNotFoundException;
+import model.Product;
+import service.ProductManager;
+
+public class Main {
+    public static void main(String[] args) {
+        ProductManager productManager = new ProductManager();
+
+        try {
+            productManager.addProduct(new Product(1, "Laptop", 999.99, 10));
```

Comment:
we needed to have a gui so the user can choose the type of function and input data. please change it so user can input and choose from console.

Reply:
Done